src/posthog-core.ts

```diff
@@ -989,6 +994,9 @@ export class PostHog {
             properties['$duration'] = parseFloat((duration_in_ms / 1000).toFixed(3))
         }
 
+        properties['$browser_type'] =
+            userAgent && _isBlockedUA(userAgent, this.config.custom_blocked_useragents) ? 'bot' : 'browser'
+
```

Comment:
nit picking for the future traveller I wonder if we should only add this prop if there is a userAgent.

I think it should always be present but JS has surprised me before now 🤣

Reply:
Actually yeah, some folk do use this in environments with no `window` (even though it's very shaky when they do) in which case we'd default to browser when we should omit it

Reply:
@pauldambra updated - thanks for the suggestion